python/get_python.bat

```diff
@@ -59,8 +57,19 @@ if ERRORLEVEL 1 (
     EXIT /b 1
 )
 
+:install_packages
 echo calling PIP to install requirements...
 call "%CMD_DIR%\pip.cmd" install -r "%CMD_DIR%/requirements.txt" --disable-pip-version-check --no-warn-script-location
+if ERRORLEVEL 1 (
```

Comment:
Pip could return [other error codes](https://github.com/pypa/pip/blob/main/src/pip/_internal/cli/status_codes.py) so this may want to use:
`if ERRORLEVEL NEQ 0`

Reply:
```
if ERRORLEVEL 1
```
actually means 1 or greater.  Its true for 1, 2, 3,... MAX_INT